src/screens/CandidateDetail/CandidatePreview.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Colors from '../../styles/colors';
+import Mixins from '../../styles/mixins';
+import PropTypes from 'prop-types';
+import {StyleSheet, TouchableOpacity, View, ScrollView, Text, FlatList } from 'react-native';
+import { Avatar,Icon } from 'react-native-elements';
+
+const FavoritesScreen = props => (
+  <View style={styles.container}>
+    <FlatList
+      data={props.data}
+      keyExtractor={(item)=>item.id}
+      renderItem={({item})=> (
+        <View style={styles.candidateContainer} >
+          <Text style={styles.nameText} >{item.name}</Text>
+          <Text style={styles.positionsText}>Running for {item.electionIds} </Text>
+          <ScrollView horizontal={true}>
+            <View style={styles.container} >
+              <View style={styles.newsCard}>  
+                <TouchableOpacity onPress={props.goToCandidateDetail(item.id)}> 
+                  <Avatar xlarge round={false} source={{ uri: item.image}}/>
+                </TouchableOpacity>
+                <View style={styles.newsBody}>
+                  <Text style={styles.newsBodyText}>
+                    Does Gavin Newsom represent a shift in California Democratic Party?
```

Comment:
In the future, it would probably be better to put your hardcoded data at the top of the file in a props object so that it will be easier to swap out for the real props once it is connected to the database